compiler/rustc_feature/src/lib.rs

```diff
@@ -108,8 +108,6 @@ impl UnstableFeatures {
 
 fn find_lang_feature_issue(feature: Symbol) -> Option<NonZeroU32> {
     if let Some(info) = ACTIVE_FEATURES.iter().find(|t| t.name == feature) {
-        // FIXME (#28244): enforce that active features have issue numbers
-        // assert!(info.issue.is_some())
```

Comment:
Should we keep the assert uncommented?

Reply:
I tried but that ICEd because it hit so called "internal" features that does not have tracking issues (and those features are special cased by the tidy check )